receiver/elasticapmintakereceiver/testdata/errors_expected.yaml

```diff
@@ -160,102 +160,6 @@ resourceLogs:
         - key: labels.organization_uuid
           value:
             stringValue: 9f0e9d64-c185-4d21-a6f4-4673ed561ec8
-        - key: http.version
-          value:
-            stringValue: "1.1"
-        - key: http.request.headers
```

Comment:
Most of these fields are removed since they are the elastic specific fields that are already set in the log record